src/testing/weather.test.js

```diff
@@ -0,0 +1,25 @@
+describe('Selectors in weatherSlice', () => {
+  const testData = {
+    weather: [
+      { main: { temp: 27.44 } },
+      { name: 'Pakistan' },
+    ],
+    Status: false,
+    error: null,
+  };
+
+  it('Weather should return the weather data', () => {
+    const selectedData = testData.weather;
+    expect(selectedData).toEqual(testData.weather);
+  });
+
+  it('MockState should return the weather status', () => {
+    const selectedStatus = testData.Status;
+    expect(selectedStatus).toBe(testData.Status);
+  });
+
+  it('MockState should return the weather error', () => {
+    const selectedError = testData.error;
+    expect(selectedError).toBe(testData.error);
+  });
```

Comment:
- [ ] Currently, these tests don't test any of the project's methods or functionalities. Kindly add more integration tests and tests that simulate user interaction, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests,the%20Redux%20Store.) requirement.